lib/on_membership_event.js

```diff
@@ -55,7 +55,9 @@ function createSetHandler(ringpop) {
                 ringpop.suspicion.start(update);
             }
 
-            ringpop.dissemination.recordChange(update);
+            if(update.address === ringpop.whoami()) {
```

Comment:
Also.. I actually don't think this is necessary. I believe that the 'local' member is added to the dissemination component before the membership set event is fired. Therefore, we won't need to `recordChange()` at all in the event handler. We might be able to kill this line altogether and achieve the same.